assets/shaders/compute_mesh.wgsl

```diff
@@ -0,0 +1,75 @@
+// This shader is used for the compute_mesh example
+// The actual work it does is not important for the example and
+// has been hardcoded to return a cube mesh
+
+struct FirstIndex {
+    vertex: u32,
+    vertex_index: u32,
+}
+
+@group(0) @binding(0) var<uniform> first_index: FirstIndex;
+@group(0) @binding(1) var<storage, read_write> vertex_data: array<f32>;
+@group(0) @binding(2) var<storage, read_write> index_data: array<u32>;
+
+@compute @workgroup_size(1)
+fn main(@builtin(global_invocation_id) global_id: vec3<u32>) {
+    for (var i = 0u; i < 192; i++) {
+        // buffer is bigger than just our mesh, so we use the first_index.vertex
```

Comment:
This needs more explanation (Bevy suballocates space for each individual mesh within a larger shared buffer (and actually multiple buffers)).